docs/src/components/Layout/SecondaryNav.tsx

```diff
@@ -0,0 +1,106 @@
+import Link from 'next/link';
+import { Heading } from '@aws-amplify/ui-react';
+import { useRouter } from 'next/router';
+
+const NavLink = ({ href, children }) => {
+  const { pathname } = useRouter();
+  const isCurrent = pathname === href;
+
+  return (
+    <Link href={href}>
+      <a className={`docs-secondary-nav-link ${isCurrent ? 'current' : ''}`}>
```

Comment:
Why doesn't this result in nested `a` tags?

Reply:
The way Next's Link works: https://nextjs.org/docs/api-reference/next/link